client/my-sites/site-settings/jetpack-module-toggle/style.scss

```diff
@@ -1,7 +1,7 @@
 .jetpack-module-toggle .form-toggle__label *:not( .form-toggle__switch ) {
-	margin-left: 8px;
+	margin-left: 12px;
```

Comment:
Can this just be removed, if we're resetting it to the same as the general toggle spacing?